src/app/components/README.md

```diff
@@ -1,11 +0,0 @@
-# Components
-
-## Overview
```

Comment:
```suggestion
We are now creating new components within [Psammead] (https://github.com/bbc/psammead), our component library. We are in the process of migrating all existing components out of Simorgh and into Psammead. If you need to add a component to be used in this application, please do so within Psammead. 
```

Reply:
Will remove them here then have another PR to add a very simple readme to the base of every major dir

Reply:
https://github.com/bbc/simorgh/issues/2873